### ISSUE TITLE
QuestionHelper input breaks UTF-8 characters

### ISSUE THREAD

**Issue by Bast1onCZ** [symfony/symfony#37495]
**Symfony version(s) affected**: 5.1.0

**Description**  
When an UTF-8 character is confirmed in a console as a response to QuestionHelper's question, it gets replaced by `'\'` or `''`.
E.g. `'Čeština'` > `'e\tina'`. My OS is Windows 10.

**How to reproduce**  
```
// inside Command::execute

$response = $questionHelper->ask($input, $output,
    new Question("Write something containing UTF-8 characters")
);
// response contains UTF-8 characters replaced by '\' or removed
```

**Comment by chalasr** [comment]
On which OS/Terminal are you experiencing this?

**Comment by Bast1onCZ** [comment]
> On which OS/Terminal are you experiencing this?

It's Windows 10. 

I'd like also note, that when passing UTF-8 characters to commands, it works properly, so there must be something wrong with question utilities

**Comment by chalasr** [comment]
Looks like it is the same problem as reported in #37278, should be fixed by #37385.
